src/netbase.cpp

```diff
@@ -38,7 +37,7 @@ static const int SOCKS5_RECV_TIMEOUT = 20 * 1000;
 static std::atomic<bool> interruptSocks5Recv(false);
 
 enum Network ParseNetwork(std::string net) {
-    boost::to_lower(net);
+    std::transform(net.begin(), net.end(), net.begin(), [](unsigned char c) -> unsigned char {return std::tolower(c);});
```

Comment:
You can pass `tolower` directly like so: `std::transform(net.begin(), net.end(), net.begin(), tolower);`

The same doesn't work with `std::tolower` fyi.

Reply:
You could cast like so, I believe that has the same effect: `std::transform(net.begin(), net.end(), net.begin(), (int (*)(int)) std::tolower);`